Lib/idlelib/squeezer.py

```diff
@@ -285,7 +287,7 @@ def count_lines(self, s):
         """
         return count_lines_with_wrapping(s, self.editwin.width)
 
-    def squeeze_current_text_event(self, event):
+    def squeeze_current_text(self):
         """squeeze-current-text event handler
```

Comment:
Start with verb, end line with period.
```suggestion
        """Handle <<squeeze-current-text>> pseudoevent.
```
Brackets could be left off.

Reply:
Done.